specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/stable/2021-04-01/DiskMigrationJobs.json

```diff
@@ -0,0 +1,471 @@
+          },
+          {
+            "$ref": "DiskMigrationJobs.json#/parameters/MigrationIdParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The disk migration job has been returned.",
+            "schema": {
+              "$ref": "DiskMigrationJobs.json#/definitions/DiskMigrationJob"
+            }
+          }
+        }
+      },
+      "put": {
+        "x-ms-examples": {
+          "Create a disk migration job.": {
+            "$ref": "./examples/DiskMigrationJobs/Create.json"
+          }
+        },
+        "tags": [
+          "DiskMigrationJobs"
+        ],
+        "operationId": "DiskMigrationJobs_Create",
+        "description": "Create a disk migration job.",
+        "parameters": [
+          {
+            "$ref": "Disks.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "DiskMigrationJobs.json#/parameters/MigrationIdParameter"
+          },
+          {
+            "$ref": "DiskMigrationJobs.json#/parameters/TargetShareParameter"
+          },
+          {
+            "$ref": "DiskMigrationJobs.json#/parameters/TargetScaleUnitParameter"
+          },
+          {
+            "$ref": "DiskMigrationJobs.json#/parameters/TargetVolumeLabelParameter"
+          },
+          {
+            "$ref": "DiskMigrationJobs.json#/parameters/DiskListParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK --  Disk migration job is created successfully.",
+            "schema": {
+              "$ref": "DiskMigrationJobs.json#/definitions/DiskMigrationJob"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute.Admin/locations/{location}/diskMigrationJobs/{migrationId}/cancel": {
+      "post": {
+        "x-ms-examples": {
+          "Cancel a disk migration job.": {
+            "$ref": "./examples/DiskMigrationJobs/Cancel.json"
+          }
+        },
+        "tags": [
+          "DiskMigrationJobs"
+        ],
+        "operationId": "DiskMigrationJobs_Cancel",
+        "description": "Cancel a disk migration job.",
+        "parameters": [
+          {
+            "$ref": "Disks.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "DiskMigrationJobs.json#/parameters/MigrationIdParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
```

Comment:
>"default" [](http://example.com/codeflow?start=10&length=9)

Please review your other APIs, you must include a default response in all your API calls to handle service failures. See [here](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html#common-issues-found-in-review)